website/docs/r/rest.html.markdown

```diff
@@ -28,9 +29,9 @@ resource "mso_rest" "sample_rest" {
     }
```

Comment:
Can we also refer tenantId to should refer to mso_tenant.tenant1.id ?

Reply:
ok